src/pseudopeople/interface.py

```diff
@@ -162,7 +162,7 @@ def generate_decennial_census(
     source: Union[Path, str] = None,
     seed: int = 0,
     config: Union[Path, str, Dict[str, Dict]] = None,
-    year: int = 2020,
+    year: Union[int, None] = 2020,
```

Comment:
You can do `Optional[int]`. This is equivalent to `Union[int, None]` you did and the preferred way to type this. Same for each function.

Reply:
In this case, we want to default to 2020, so I don't think we want `Optional[int]`. https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
Nevermind, this is good suggestion, updated.